src/Http/Controllers/Settings/EntryTypesController.php

```diff
@@ -89,6 +89,10 @@ public function edit(Request $request, ?int $entryTypeId = null): CpScreenRespon
     {
         $entryTypeId ??= $request->input('entryTypeId');
```

Comment:
We could use `$request->integer`, it returns `0` instead of `null` in case it wasn’t passed, that way we don’t have to manually cast it

Reply:
That said, I made a few stupid errors here, so let me fix those first 🤦‍♀️ 

We can discuss the preferred approach on Monday and go from there.